build.sbt

```diff
@@ -30,18 +30,17 @@ val sonatypeReleaseSettings = Seq(
     "scm:git:git@github.com:guardian/play-googleauth.git"
   )),
 
-  releasePublishArtifactsAction := PgpKeys.publishSigned.value,
```

Comment:
I think this particular line is superfluous, as `releaseStepCommandAndRemaining("+publishSigned")` takes care of it?

It's not mentioned in https://github.com/xerial/sbt-sonatype#using-with-sbt-release-plugin